helm/annotations-rw-neo4j/templates/deployment.yaml

```
@@ -63,6 +63,8 @@ spec:
               key: msk.kafka.broker.url
         - name: LIFECYCLE_CONFIG_PATH
           value: {{ .Values.env.LIFECYCLE_CONFIG_PATH }}
+        - name: API_HOST
+          value: http://api.ft.com
```

`question` non-blocking: Should we use`https` here?

---

We can, but we use `http` everywhere else and that's why I've kept `http`.

My initial plan was to use:

```suggestion
          valueFrom:
            configMapKeyRef:
              name: global-config
              key: api.host.with.protocol
```

This would result in `https://api-t.ft.com` for dev & staging and `https://api.ft.com` for prod, but I left this change for the future.

---

I don't think it's a big deal for the GET response if this service(as I don't believe there are users of it) but overall if we are returning `http://api.ft.com` so far for the thing url, we should keep it as that. Otherwise, clients may be affected.